packages/perseus/src/util/answer-types.ts

```diff
@@ -625,11 +624,11 @@ const KhanAnswerTypes = {
         convertToPredicate: function (
             correctAnswer: string,
             options: any,
-        ): Predicate {
```

Comment:
This isn't returning a `Predicate` by itself. It's returning a tuple of two items (one being the Predicate and the other the options).